src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -789,6 +789,24 @@ def pickRSE(self, rseExpression='rse_type=TAPE\cms_type=test', rseAttribute='ddm
         choice = weightedChoice(rsesWithWeights)
         return choice
 
+    def requiresApproval(self, rse):
+        """
+        _requiresApproval_
+
+        Returns wether or not the specified RSE requires rule approval.
+        :param res: Rucio RSE name
+        :returns: True if the RSE requires approval to write (rule property)
+        """
+        try:
+            attrs = self.cli.list_rse_attributes(rse)
+        except RSENotFound as exc:
+            msg = "Error retrieving attributs for RSE: {}. Error: {}".format(rse, str(exc))
+            raise WMRucioException(msg)
```

Comment:
I'd replace the else: ... by something like:
```
return attrs.get('requires_approval', False)
```

if we got to this point, then there was no exception.